lumigator/python/mzai/sdk/sdk/experiments.py

```diff
@@ -1,77 +0,0 @@
-from http import HTTPMethod, HTTPStatus
-from json import dumps
-from uuid import UUID
-
-from schemas.experiments import (
-    ExperimentCreate,
```

Comment:
I wouldn't get rid of experiments just yet.
On the backend side it's a dummy right now, but the intention is to use to to trigger a very specific workflow/combination of jobs, that is, the 2 that used to be the "old" experiments:

One of:
* evaluation job, assuming there are predictions and labels (aka ground_truth)
or
* inference (ground_truth generation) job, evaluation job (assuming we only have predictions)

In the future, it would make most sense to generate the predictions as well.

None of this is in main, though, and experiemnts right now just calls jobs, so at least the boilerplate with a get for a list of experiemnts is worth keeping (I think).

Reply:
Ok. I'll keep the experiments endpoints just in case.